### PR TITLE
Last small refactoring before storm

### PR DESCRIPTION
**Pull request by aplanas**
This also fix a bug in the `apiurl`

### CODE REVIEW

File: git-importer.py [hunk 1]
```diff
@@ -151,14 +151,9 @@ def get_revisions(self, project):
...
-        for rev in self.projects.get(project, []):
-            if rev.time > time:
-                return prev
-            if rev.time == time:
-                return rev
-            prev = rev
-        return prev
+        revs = self.projects.get(project, [])
+        revs = sorted(revs, key=lambda x: x.time)
+        return revs[0] if revs else None
```
Comment (coolo) [inline git-importer.py]: I don't think that's equal as in ignoring the time argument alltogether

Comment (aplanas) [reply git-importer.py]: The "time" is in the line 155: `x.time`. This line can be read as "order the list based on the `time` attribute"

Comment (coolo) [reply git-importer.py]: I know, but the function you're replacing did more and while your implementation matches the function name, the removed code is more `find_last_revision_after_time`. The revisions *are* sorted after time already (unless someone invented a time machine and managed to rewrite history, there is other way than that with obs) - so this is looking through them for the revision matching the given time parameter.

Comment (coolo) [reply git-importer.py]: You can easily verify your code being broken by trying to import e.g. zstd

```
ERROR:root:package can't be expanded in Rev Archiving/60 Md5 9ee40c7b9739b7721c99eff07948eddf 2021-05-11 20:29:41 bmwiedemann None
commit Archiving 60
ERROR:root:package can't be expanded in Rev Archiving/61 Md5 7dad5b0af7add6491d8e908ea858db46 2021-05-11 20:30:30 bmwiedemann None
commit Archiving 61
ERROR:root:package can't be expanded in Rev Archiving/62 Md5 321139c8ed49e77cd58c61b2730f6345 2021-05-14 22:44:49 namtrac None
commit Archiving 62
ERROR:root:package can't be expanded in Rev Archiving/63 Md5 21ab841b1de676a94d5dced18b4743d3 2021-05-14 22:55:08 namtrac None
commit Archiving 63
ERROR:root:package can't be expanded in Rev Archiving/64 Md5 7e58267908d99ad26fb3a3c049a4d916 2021-05-14 22:56:04 namtrac None
commit Archiving 64
ERROR:root:package can't be expanded in Rev Archiving/65 Md5 368063780332941279b265d44527ad07 2021-05-14 22:56:40 namtrac None
commit Archiving 65
ERROR:root:package can't be expanded in Rev Archiving/66 Md5 593539cebda8c0ef3cab4fb3ca33d24f 2021-05-14 23:12:38 namtrac None
commit Archiving 66
ERROR:root:package can't be expanded in Rev Archiving/67 Md5 bf048d95501a037bb2a037411427d445 2021-05-14 23:14:24 namtrac None
commit Archiving 67
```

**Pull request merged**